onboarding/source/amb_light_service.c

```diff
@@ -17,37 +21,126 @@
 
 /* USER CODE BEGIN */
 // Define light service queue config here
+#define QUEUE_LENGTH 10
+#define ITEM_SIZE sizeof(uint64_t)
```

Comment:
The items sent to the queue have a type of `light_event_t`. You should use `sizeof(light_event_t)` instead